ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage  extends Page {
+
+    private  HomePage homePage;
+    private static String name = "john";
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By NAME = By.cssSelector("button, input");
+    private static final By NEXT_BUTTON = By.id("next-button");
+    private static final By CONFIRM_BUTTON = By.id("registration-submit");
+    
+
+    public EditPatientRelationshipPage(Page parent) {
+        super(parent);
+    }
+
+    public void clickOnSelectRelationshipType() throws InterruptedException{
+        clickOn(SELECT_RELATIONSHIP_TYPE);
+        Select relationshipType = new Select(driver.findElement(By.id("relationship_type")));
+        relationshipType.selectByVisibleText("Doctor");
+        setTextToFieldNoEnter(NAME,name);
+        clickOn(NEXT_BUTTON);
```

Comment:
before this, where is the step that selects `John` from the popup with results?

Reply:
setTextToFieldNoEnter(NAME,name); allows the input of the string "john"